packages/ui/src/composables/useSelectable.ts

```diff
@@ -0,0 +1,143 @@
+import { PropType, computed, SetupContext, Ref } from 'vue'
```

Comment:
How about to move this hook to checkbox if it used only in VaCheckbox? See TimePicker.